---
title: BGA23 Sessions
description: Current list of sessions at Biodiversity Genomics Academy 2023
---

## Planned Sessions

The list of sessions planned so far is given below. We are currently in the process of finalising the dates and times of these **online** sessions between 1-30 September 2023. 

| Title | Session Leader |
|-------|----------------|
| Planning genome projects using [GoaT](https://goat.genomehubs.org) | Cibele Sotero-Caio |
| Understanding k-mers and ploidy using [Smudgeplots](https://github.com/KamilSJaron/smudgeplot) | Kamil Jaron |
| Assembling Mito genomes using [MitoHifi](https://github.com/marcelauliano/MitoHiFi) | Marcela Uliano-Silva |
| Checking for cobionts in public genomes using [BlobToolKit](https://blobtoolkit.genomehubs.org/view) | Sujai Kumar |
| Visualising genome assembly cobionts by running [BlobToolKit](https://blobtoolkit.genomehubs.org/view) | Sujai Kumar |
| Using the [Vertebtate Genome Project (VGP) Pipeline in Galaxy](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/vgp_genome_assembly/tutorial.html) | Delphine Lariviere |
| Searching and Downloading Genome Data using [NCBI Datasets](https://www.ncbi.nlm.nih.gov/datasets/) | Nuala O'Leary |
| Curating genomes using [TreeVal](https://github.com/sanger-tol/treeval) | Genome Reference Informatics Team |
| Annotating genomes the [Ensembl](https://rapid.ensembl.org/info/genome/genebuild/index.html) way | Leanne Haggerty and José G Pérez-Silva |
| Classifying Transposable Elements with [Earl Grey TE](https://github.com/TobyBaril/EarlGrey) | Tobias Baril |
| Exploring and retrieving open biodiversity data using EMBL-EBI's biodiversity portals | Joana Pauperio |

## Proposed Sessions

In addition, we would like to host sessions on some of the following topics as well. So if you are a developer or an expert in any of these, and are willing to lead a session on any of the following, please contact [sujai.kumar@sanger.ac.uk](mailto:sujai.kumar@sanger.ac.uk) or the social media accounts below.

This is not an exhaustive list, so any other related topics are also welcome. If there are enough volunteers we will continue the sessions outside the Biodiversity Genomics conference schedule beyond September as well.

| Topic |
|-------|
| Introduction to HiFi data (theory and practice) |
| Introduction to Hi-C data (theory and practice) |
| Introduction to NextFlow for genomics |
| Quality control of raw genomic data |
| VGP Raw data QC toolkit in Galaxy |
| VGP Primary assembly toolkit in Galaxy |
| Purging haplotypic duplication from assemblies (theory and practice) |
| Understanding genome characteristics using GenomeScope 2.0 |
| From genome to phylogenetic gene set with BUSCO |
| Using ENA genomics resources |
| OMA and OMARK for genome completeness exploration |
| Assembly with HiFiASM: “normal”, “Hi-C” and metagenome modes |
| YAHS and Hi-C scaffolding |
| Interpreting Hi-C plots in quality assessment and curation |
| Using Bandage to explore assembly graphs |
| Genefinding with BRAKER3 |
| Verkko: telomere-to-telomere assembly of diploid chromosomes |
| Using Merqury to assess assemblies |
| Using OrthoFinder to find orthologs across genomes |
| Aligning whole genomes using Cactus |
| Using the Telomere Identification toolKit (tidk) |
| Using the Organelle Assembly ToolKit (oatk) |

## How do I sign up?

Be the first to be informed as soon as registrations open for these free and open to all sessions by signing up for our low-volume mailing list. We will use this list ONLY for Biodiversity Genomics Academy information. You can unsubscribe at any time.

[Subscribe](https://zcmp.eu/QIAZ){ .md-button .md-button--primary }

In addition, please bookmark this website, and follow our social media accounts:

:fontawesome-brands-twitter: [twitter.com/BGAcademy23](https://twitter.com/BGAcademy23)

:fontawesome-brands-mastodon: [genomic.social/@BGAcademy23](https://genomic.social/@BGAcademy23)<a rel="me" href="https://genomic.social/@BGAcademy23"></a>
