---
title: Sessions
description: Current list of sessions at Biodiversity Genomics Academy 2023
---

## Planned Sessions

Scheduled sessions are now open for registration. New sessions and repeated sessions will be added to this page, so please keep checking back or [sign up](signup.md) to be emailed when new/repeat sessions become available.

Detailed session contents will be added over time at this site. Instructions for joining each session will be sent to registered participants.

If there are 2 time slots available for a session, pick the one you can attend more conveniently [in your time zone](https://www.timeanddate.com/worldclock/meetingtime.html?iso=20230915&p1=224&p2=41&p3=45&p4=4&p5=37&p6=170&p7=176&p8=33&p9=240&p10=22){: target="_blank"}.

??? info "**2023-08-07 Update** (Click to Expand)"
    The response to BGA23 has been OVERWHELMING! Thank you üôè **691 booked, 424 waitlisted** ü§Ø Please keep signing up below as we are working with session leaders to add more slots (adding sessions, increasing slots per session, offering recordings with discord discussions, etc). Booked out sessions have a "Join Waitlist" button. Also do **[subscribe/follow](signup.md)** to be the first to be informed when slots open.

??? info "**2023-08-11 Update** (Click to Expand)"
    We are notifying waitlisted participants today by email on how to book extra slots. If you have a confirmed slot already that you can no longer attend, please email tol-training at sanger.ac.uk with your order details so we can free up your slot for other waitlisted participants. Thank you!

    Sessions added/changed:

    - "Introduction to manual curation - Treeval" renamed to "Setting up the Treeval pipeline for generating evidence for manual curation", and a new session added 18 Sep 11:00-13:00
    - "Starting a comparative genome study from [CNGBdb](https://db.cngb.org/){: target="_blank"}: datasets, analysis platform, and spatial transcriptomics database" new session 13 Sep 09:00-11:00
    - "Genome profiling using [Genomescope](https://github.com/tbenavi1/genomescope2.0){: target="_blank"} new session 15 Sep 09:00-11:00
    - "[OMA](https://omabrowser.org/oma/home/){: target="_blank"} and [OMArk](https://omark.omabrowser.org/){: target="_blank"} for homology exploration and gene annotation quality control" new session 15 Sep 16:00-18:00
    - "Introduction to Manual Curation - HiC and JBrowse" new session added 19 Sep 11:00-13:00
    - "Introduction to NextFlow for Genomics" two new sessions added 20 Sep 23:00-01:00, and 26 Sep 16:00-18:00
    - "[EASEL](https://gitlab.com/PlantGenomicsLab/easel){: target="_blank"} (Efficient, Accurate, Scalable Eukaryotic modeLs): software for eukaryotic structural and functional genome annotation" new session added 29 Sep 03:00-05:00

??? info "**2023-08-14 Update** (Click to Expand)"
    We are up to **1167 sign ups and 793 waitlisted slots**!
    
    Seeing the huge demand, some session leaders have very kindly offered webinar slots (view only + text discussions) for:

    - Assembling Genomes with the VGP-Galaxy Pipeline
    - Classifying Transposable Elements with Earl Grey TE
    - Assess genome assemblies with Merqury
    - Annotating genomes the Ensembl way - General concepts + Case study
    - Annotating genomes the Ensembl way: Hands on - From RNAseq reads to gene models

    If you signed up to a session waitlist after Friday 11 Aug, you will receive information on additional slots by the end of today.

??? info "**2023-08-16 Update** (Click to Expand)"
    We are up to **1475 sign ups and 1018 waitlisted slots**!

    All waitlisted slots as of 4pm UTC should have received information about additional slots. After this point, the only free slots are likely to be from cancellations. A couple of new topics may be added, but no additional slots are likely to be opened for existing topics.

    To be fair to everyone, we will be asking everyone to confirm attendance for each session by next week, so please keep an eye out for those emails.

!!! info "**2023-08-25 Update**"
    We are up to **1680 sign ups and 1273 waitlisted slots**!

    Sessions added/changed:

    - "Gene prediction with [BRAKER3](https://github.com/Gaius-Augustus/BRAKER){: target="_blank"}" - new sessions added 22 Sep 08:00-10:00 and 25 Sep 12:00-14:00
    - "[OMA](https://omabrowser.org/oma/home/){: target="_blank"} and [OMArk](https://omark.omabrowser.org/){: target="_blank"} for homology exploration and gene annotation quality control" new session added 21 Sep 08:00-10:00


| Title | Session Leader | Date and Time in UTC/GMT | 
|-------|----------------|--------------------------|
| Checking for cobionts in public genomes using [BlobToolKit](https://blobtoolkit.genomehubs.org/view){: target="_blank"} | Sujai Kumar | [Register](https://buytickets.at/bga23/973209){ : target="_blank" .md-button .md-button--primary }<br/>4 Sep 08:00-09:00<br/>4 Sep 20:00-21:00 |
| Querying genome metadata and sequencing projects using [GoaT](https://goat.genomehubs.org){: target="_blank"} | Cibele Sotero-Caio | [Register](https://buytickets.at/bga23/973002){ : target="_blank" .md-button .md-button--primary }<br/>6 Sep 16:00-18:00 |
| The [TreeVal Pipeline](https://github.com/sanger-tol/treeval){: target="_blank"}: Generating evidence for manual curation| Genome Reference Informatics Team, Sanger | [Register](https://buytickets.at/bga23/973202){ : target="_blank" .md-button .md-button--primary }<br/>8 Sep 11:00-13:00<br/>18 Sep 11:00-13:00 |
| Assembling Mitogenomes from PacBio HiFi reads using [MitoHiFi](https://github.com/marcelauliano/MitoHiFi){: target="_blank"} | Marcela Uliano-Silva | [Register](https://buytickets.at/bga23/973171){ : target="_blank" .md-button .md-button--primary }<br/>8 Sep 13:00-15:00 | 
| Visualising genome assembly cobionts by running [BlobToolKit](https://blobtoolkit.genomehubs.org/view){: target="_blank"} locally | Sujai Kumar | [Register](https://buytickets.at/bga23/973213){ : target="_blank" .md-button .md-button--primary }<br/>11 Sep 08:00-09:00<br/>11 Sep 20:00-21:00 |
| Understanding k-mers and ploidy using [Smudgeplot](https://github.com/KamilSJaron/smudgeplot){: target="_blank"} | Kamil Jaron | [Register](https://buytickets.at/bga23/973117){ : target="_blank" .md-button .md-button--primary }<br/>12 Sep 14:00-16:00 |
| Starting a comparative genome study from [CNGBdb](https://db.cngb.org/){: target="_blank"}: datasets, analysis platform, and spatial transcriptomics database  | Xiaofeng Wei, Huanming Yang, Xin Liu | [Register](https://buytickets.at/bga23/980802){ : target="_blank" .md-button .md-button--primary }<br/>13 Sep 09:00-11:00 |
| Searching and Downloading Genome Data using [NCBI Datasets](https://www.ncbi.nlm.nih.gov/datasets/){: target="_blank"} | Nuala O'Leary | [Register](https://buytickets.at/bga23/973214){ : target="_blank" .md-button .md-button--primary }<br/>14 Sep 17:00-18:00 | 
| Compare whole genome assembly alignments using NCBI‚Äôs [Comparative Genome Viewer (CGV)](https://www.ncbi.nlm.nih.gov/genome/cgv){: target="_blank"} | Sanjida Rangwala | [Register](https://buytickets.at/bga23/973216){ : target="_blank" .md-button .md-button--primary }<br/>14 Sep 18:00-19:00 |
| Genome profiling using [Genomescope](https://github.com/tbenavi1/genomescope2.0){: target="_blank"} | Luc√≠a Campos-Dom√≠nguez and Kamil Jaron | [Register](https://buytickets.at/bga23/981915){ : target="_blank" .md-button .md-button--primary }<br/>15 Sep 09:00-11:00 |
| Introduction to Manual Curation - HiC and JBrowse | Genome Reference Informatics Team, Sanger | [Register](https://buytickets.at/bga23/973204){ : target="_blank" .md-button .md-button--primary }<br/>15 Sep 11:00-13:00<br/>19 Sep 11:00-13:00 |
| [OMA](https://omabrowser.org/oma/home/){: target="_blank"} and [OMArk](https://omark.omabrowser.org/){: target="_blank"} for homology exploration and gene annotation quality control | Yannis Nevers and Sina Majidian | [Register](https://buytickets.at/bga23/981722){ : target="_blank" .md-button .md-button--primary }<br/>15 Sep 16:00-18:00<br/>21 Sep 08:00-10:00 |
| Introduction to NextFlow for Genomics | Solenne Correard | [Register](https://buytickets.at/bga23/973982){ : target="_blank" .md-button .md-button--primary }<br/>19 Sep 19:00-21:00<br/>20 Sep 23:00-01:00<br/>26 Sep 16:00-18:00 |
| [BUSCO](https://busco.ezlab.org/){: target="_blank"} - from QC to gene prediction and phylogenomics | Rob Waterhouse | [Register](https://buytickets.at/bga23/973174){ : target="_blank" .md-button .md-button--primary }<br/>20 Sep 08:00-10:00<br/>20 Sep 17:00-19:00 |
| Classifying Transposable Elements with [Earl Grey TE](https://github.com/TobyBaril/EarlGrey){: target="_blank"} | Tobias Baril | [Register](https://buytickets.at/bga23/973218){ : target="_blank" .md-button .md-button--primary }<br/>21 Sep 12:00-14:00 |
| Assembling Genomes with the [VGP-Galaxy Pipeline](https://training.galaxyproject.org/training-material/topics/assembly/tutorials/vgp_genome_assembly/tutorial.html){: target="_blank"} | Delphine Lariviere | [Register](https://buytickets.at/bga23/973190){ : target="_blank" .md-button .md-button--primary }<br/>21 Sep 17:00-19:00 |
| Gene prediction with [BRAKER3](https://github.com/Gaius-Augustus/BRAKER){: target="_blank"} | Katharina Hoff | [Register](){ : target="_blank" .md-button .md-button--primary }<br/>22 Sep 08:00-10:00<br/>25 Sep 12:00-14:00 |
| [Gfastar](https://github.com/vgl-hub/gfastar): a tool suite to assist genome assembly | Giulio Formenti and Cassidy Johnson | [Register](https://buytickets.at/bga23/973193){ : target="_blank" .md-button .md-button--primary }<br/>22 Sep 14:00-16:00 |
| Scalable telomere-to-telomere assembly with [hifiasm](https://github.com/chhylp123/hifiasm){: target="_blank"} | Haoyu Cheng | [Register](https://buytickets.at/bga23/975629){ : target="_blank" .md-button .md-button--primary }<br/>22 Sep 17:00-18:00 |
| ENA Standards and Submission | Maira Ihsan | [Register](https://buytickets.at/bga23/973225){ : target="_blank" .md-button .md-button--primary }<br/>25 Sep 09:00-11:00<br/>26 Sep 18:00-20:00 |
| ENA data retrieval and EMBL-EBI's Biodiversity Portals | Alexey Sokolov | [Register](https://buytickets.at/bga23/973223){ : target="_blank" .md-button .md-button--primary }<br/>26 Sep 09:00-10:00<br/>27 Sep 18:00-19:00 |
| Assess genome assemblies with [Merqury](https://github.com/marbl/merqury){: target="_blank"} | Arang Rhie | [Register](https://buytickets.at/bga23/973198){ : target="_blank" .md-button .md-button--primary }<br/>26 Sep 12:00-14:00<br/>28 Sep 20:00-22:00 |
| [EASEL](https://gitlab.com/PlantGenomicsLab/easel){: target="_blank"} (Efficient, Accurate, Scalable Eukaryotic modeLs): software for eukaryotic structural and functional genome annotation | Jill Wegrzyn and Cynthia Webster | [Register](https://buytickets.at/bga23/975736){ : target="_blank" .md-button .md-button--primary }<br/>27 Sep 13:00-15:00<br/>29 Sep 03:00-05:00 |
| Annotating genomes the [Ensembl](https://rapid.ensembl.org/info/genome/genebuild/index.html){: target="_blank"} way: General Concepts and Background + A Case study: Annotating Fungal Genomes - The Challenges! | Leanne Haggerty | [Register](https://buytickets.at/bga23/976019){ : target="_blank" .md-button .md-button--primary }<br/>28 Sep 12:00-14:00 |
| Annotating genomes the [Ensembl](https://rapid.ensembl.org/info/genome/genebuild/index.html){: target="_blank"} way: Hands-on - From RNAseq reads to gene models | Jos√© G P√©rez-Silva | [Register](https://buytickets.at/bga23/976020){ : target="_blank" .md-button .md-button--primary }<br/>28 Sep 14:00-16:00 |

## Proposed Sessions

In addition, we would like to host sessions on some of the following topics as well. So if you are a developer or an expert in any of these, and are willing to lead a session on any of the following, please contact [sujai.kumar@sanger.ac.uk](mailto:sujai.kumar@sanger.ac.uk) or the social media accounts below.

This is not an exhaustive list, so any other related topics are also welcome. If there are enough volunteers we will continue the sessions outside the Biodiversity Genomics conference schedule beyond September as well.

| Topic |
|-------|
| Introduction to HiFi data (theory and practice) |
| Introduction to Hi-C data (theory and practice) |
| Introduction to Nanopore data (theory and practice) |
| Introduction to NextFlow for genomics |
| Quality control of raw genomic data |
| VGP Raw data QC toolkit in Galaxy |
| VGP Primary assembly toolkit in Galaxy |
| Purging haplotypic duplication from assemblies (theory and practice) |
| Understanding genome characteristics using GenomeScope 2.0 |
| OMA and OMARK for genome completeness exploration |
| Assembly with HiFiASM: ‚Äúnormal‚Äù, ‚ÄúHi-C‚Äù and metagenome modes |
| YAHS and Hi-C scaffolding |
| Using Bandage to explore assembly graphs |
| Genefinding with BRAKER3 |
| Verkko: telomere-to-telomere assembly of diploid chromosomes |
| Using OrthoFinder to find orthologs across genomes |
| Aligning whole genomes using Cactus |
| Using the Telomere Identification toolKit (tidk) |
| Using the Organelle Assembly ToolKit (oatk) |
